Fixes for managing content views chap (#1637)
theforeman/foreman-documentation: guides/common/modules/con_resolving-package-dependencies.adoc

[id="Resolving_Package_Dependencies_{context}"]
= Resolving Package Dependencies

> In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.

Reviewer: ```suggestion
You can resolve package dependencies to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
```

> 

Reviewer: ```suggestion
```

If you keep this empty line, each sentence would stand of itself. IMHO a general rule is that a paragraph consists of at least two sentences.

> You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.

Reviewer: Is "enabled" or "disabled" the default setting? Either way, IMHO you should remove one aka. "or you can change the default setting to disable resolving package..." or "or you can change the default setting to enable resolving package...".

Author: > Is "enabled" or "disabled" the default setting?
disable seems to be the default. changing the wording appropriately

Note that resolving package dependencies can cause significant delays to Content View promotion.
The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.

.Resolving Package Dependencies and Filters

> 

Reviewer: ```suggestion
```

Filters do not resolve any dependencies of the packages listed in the filters.
This might require some level of testing to determine what dependencies are required.

If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.

> 

Reviewer: ```suggestion
```
